framework/src/modules/chain/modules/blocks/chain.js

```diff
@@ -708,13 +692,7 @@ __private.popLastBlock = function(oldLastBlock, cb) {
 			.loadSecondLastBlockStep(oldLastBlock.previousBlock, tx)
 			.then(res => {
 				secondLastBlock = res;
-				return Promise.mapSeries(
-					oldLastBlock.transactions.reverse(),
-					transaction =>
-						__private
-							.undoConfirmedStep(transaction, oldLastBlock, tx)
-							.then(() => __private.undoUnconfirmStep(transaction, tx))
```

Comment:
Don't think we should remove undoUnconfirmStep just yet.

Reply:
Restored